src/bulma/pages/companies/components/People.vue

```diff
@@ -51,7 +51,7 @@
                 <person :id="id"
                     :person="person"
                     @edit="edit(person)"
-                    @delete="destroy(person, index)"/>
+                    @delete="deletedPerson = people[index]"/>
```

Comment:
let's rename to deletedPerson to removedPerson

also the people[index] does not seem right, maybe removedPerson = person is enough.

or we can keep the removedPersonId directly